opae.spec

```diff
@@ -138,68 +129,68 @@ DESTDIR=%{buildroot}  cmake -DCOMPONENT=toolmmlink -P cmake_install.cmake
 DESTDIR=%{buildroot}  cmake -DCOMPONENT=samplebin -P cmake_install.cmake
 DESTDIR=%{buildroot}  cmake -DCOMPONENT=libopaeheaders -P cmake_install.cmake
 DESTDIR=%{buildroot}  cmake -DCOMPONENT=safestrheaders -P cmake_install.cmake
-
-
-%clean
+DESTDIR=%{buildroot}  cmake -DCOMPONENT=toolpackager -P cmake_install.cmake
+DESTDIR=%{buildroot}  cmake -DCOMPONENT=jsonschema -P cmake_install.cmake
 
 %post
-ldconfig
-
-%postun
-ldconfig
-
-%pre
+mkdir -p %{_sysconfdir}/ld.so.conf.d
+echo "" > %{_sysconfdir}/ld.so.conf.d/opae-c.conf
 
-%preun
 
 %files
-%defattr(-,root,root,-)
 %dir %{_datadir}/opae
 %doc %{_datadir}/opae/RELEASE_NOTES.md
-%{_libdir}/libopae-c.so*
-%{_libdir}/libopae-cxx-core.so*
+%license %{_datadir}/opae/LICENSE
+%license %{_datadir}/opae/COPYING
+%{_libdir}/libopae-c.so.%{version}
+%{_libdir}/libopae-c.so.1
+%{_libdir}/libopae-c.so
+
+%{_libdir}/libbitstream.so.%{version}
+%{_libdir}/libbitstream.so.1
+%{_libdir}/libbitstream.so
+
+%{_libdir}/libopae-cxx-core.so.%{version}
+%{_libdir}/libopae-cxx-core.so.1
+%{_libdir}/libopae-cxx-core.so
+
+
 %{_libdir}/opae/libxfpga.so*
-%{_libdir}/libbitstream.so*
 %{_libdir}/opae/libmodbmc.so*
-%{_libdir}/opae/libboard_rc.so*
-%{_libdir}/opae/libboard_vc.so*
 %{_libdir}/libsafestr.a*
 
 
+
 %files devel
-%defattr(-,root,root,-)
 %dir %{_includedir}/opae
 %{_includedir}/opae/*
 %dir %{_includedir}/safe_string
 %{_includedir}/safe_string/safe_string.h
 %{_libdir}/libsafestr.a
 %dir %{_usr}/src/opae
-%{_usr}/src/opae/samples/hello_fpga.c
-%{_usr}/src/opae/samples/hello_events.c
-%{_usr}/src/opae/samples/object_api.c
+%{_usr}/src/opae/samples/hello_fpga/hello_fpga.c
+%{_usr}/src/opae/samples/hello_events/hello_events.c
+%{_usr}/src/opae/samples/object_api/object_api.c
 %{_usr}/src/opae/cmake/*
 %{_usr}/src/opae/opae-libs/cmake/modules/*
 
-
-%files tools
-%defattr(-,root,root,-)
 %{_libdir}/libargsfilter.a*
-%{_bindir}/fpgaconf*
-%{_bindir}/fpgainfo*
-
+%{_libdir}/opae/libboard_rc.so*
+%{_libdir}/opae/libboard_vc.so*
 
-%files tools-extra
-%defattr(-,root,root,-)
+%{_bindir}/fpgaconf
+%{_bindir}/fpgainfo
 %{_bindir}/mmlink
 %{_bindir}/userclk
-
-
-%files samples
-%defattr(-,root,root,-)
 %{_bindir}/hello_fpga
-%{_bindir}/hello_events*
 %{_bindir}/object_api
+%{_bindir}/hello_events
 %{_bindir}/hello_cxxcore
+%{_bindir}/afu_json_mgr
+%{_bindir}/packager
```

Comment:
we are upstreaming packager?

Reply:
including in 1.4.1 release, converting rbf file to gbs.